src/services/email.service.js

```diff
@@ -5,10 +5,17 @@ const logger = require('../config/logger');
 const transport = nodemailer.createTransport(config.email.smtp);
 /* istanbul ignore next */
 if (config.env !== 'test') {
-  transport
-    .verify()
-    .then(() => logger.info('Connected to email server'))
-    .catch(() => logger.warn('Unable to connect to email server. Make sure you have configured the SMTP options in .env'));
+  transport.verify((error, success) => {
+    switch (true) {
+      case error:
+        logger.warn('Unable to connect to email server. Make sure you have configured the SMTP options in .env');
```

Comment:
Shouldn't be `logger.error()` ? It's a broken feature in your software, so I believe it should be marked as an error in order to raise the proper attention

Reply:
I can't follow you here. What do you mean by `a broken feature in your software` ?

Reply:
Maybe change the `default` to
``` 
logger.error('transport.verify() returned unknown status');
```
or something along those lines.

Reply:
Okay. Adding to `config.email.smtp`

```javascript
  email: {
    smtp: {
      host: envVars.SMTP_HOST,
      port: envVars.SMTP_PORT,
      secure: envVars.SMTP_SECURE,
      auth: {
        user: envVars.SMTP_USERNAME,
        pass: envVars.SMTP_PASSWORD,
      },
      tls: {
        rejectUnauthorized: false,  // <== Accept selfsigned certificates
      },
    },
    from: envVars.EMAIL_FROM,
  },
```
Fixes the `transfer.verify().then().catch()` chaining problem.

I should probably close this PR and refactor.

Reply:
> I can't follow you here. What do you mean by `a broken feature in your software` ?

I just meant it was an error. You cannot send mails at that point. The `logger.error()` was all I was suggesting.

Reply:
> Okay. Adding to `config.email.smtp`
> 
> ```js
>   email: {
>     smtp: {
>       host: envVars.SMTP_HOST,
>       port: envVars.SMTP_PORT,
>       secure: envVars.SMTP_SECURE,
>       auth: {
>         user: envVars.SMTP_USERNAME,
>         pass: envVars.SMTP_PASSWORD,
>       },
>       tls: {
>         rejectUnauthorized: false,  // <== Accept selfsigned certificates
>       },
>     },
>     from: envVars.EMAIL_FROM,
>   },
> ```
> 
> Fixes the `transfer.verify().then().catch()` chaining problem.
> 
> I should probably close this PR and refactor.

So the chaining wasn't working because a configuration issue? I didn't get why it wasn't working, just assumed you were right about it not working.

Reply:
Also you should know I have no write access, so consider my revision just an opinion. I have no power to pass or refuse the PR. Just wanted to contribute my grain of salt. ;)

Reply:
You are right, it's an error that prohibit sending emails, but it was set as a warning, and I didn't want to change that.

I did make the error message more verbose, so it will tell why it's failing. That was what I was missing, trying to figure out why the `verify()` was failing no matter what. I could see the server log that a connection was made, but it failed anyway.

Anyway. Your input is appreciated. More eyes on code definitely doesn't hurt :)

I'll probably try to add certificate to the `nodemailer` config next (soonish).